src/app/src/mainwindow.cpp

```diff
@@ -230,6 +233,32 @@ void MainWindow::on_ledIntensityEdit_valueChanged(double value) {
 }
 
 
+/*
+ * Check if drive on windows has sufficient space for acquisition.
+ *
+ * @param drive letter selected in config file
+ * @param fps setting of acquisition
+ * @param duration of acquisition
+ * @param TODO
+ *
+ * @returns boolean true if space is available
+*/
+bool available_space_in_drive(std::filesystem::path driver_name, double fps,double duration,double frameBytes= 100000){
```

Comment:
You probably want to make this function a method on the MainWindow class so it has access to the `m_camera` instance